lib/mix/tasks/firmware.ex

```diff
@@ -19,31 +19,36 @@ defmodule Mix.Tasks.Firmware do
     * `NERVES_TOOLCHAIN` - may be set to a local directory to specify the
       Nerves toolchain (C/C++ crosscompiler) that is used
   """
-  def run(_args) do
+  def run(args) do
     preflight()
     debug_info("Nerves Firmware Assembler")
 
-    system_path = check_nerves_system_is_set!()
+    if "--image-path" in args do
+      config = Mix.Project.config()
+      images_path = images_path(config)
+      otp_app = config[:otp_app]
+      IO.inspect(fw_path(images_path, otp_app))
```

Comment:
Is this `IO.inspect` here on purpose? I'm a little unclear on what the purpose of this `--image-path` flag is. The docs also don't describe that this flag is here and what it does.

Reply:
I put `IO.inspect`  so that the task could be used to get the data from other Elixir code _and_ from the shell. The initial reason for this was to stop the shell script in Nerves Firmware SSH from doing this all manually. Unfortunately since Erlang spits stuff to the terminal no matter what (as far as i know), that isn't very useful. We can probably turn it to an `IO.puts` or `Mix.shell.info`. (if we decide this is actually needed that is.)

Reply:
FWIW the issue brought this up is #183